### DISCUSSION TITLE
【疑问】部署到公网后，客户端拉取资源的流是经过服务器的吗？

### DISCUSSION THREAD

**Discussion by Geolage** [1715173329/UnblockNeteaseMusic#1107] [Q&A]
### 需求描述

如题所示。因为我在部署至公网后，发现歌是能听了，但缓冲的速度很慢，请问是跟服务器有关系吗？

### 网易云音乐版本号

2.8.1

### 操作系统

Windows 11

**Comment by 1715173329** [comment]
设置 endpoint 之后网易云音乐流量就会过服务器，但是不设置的话会有兼容问题。

**Reply by Geolage** [reply]
具体是啥兼容问题呢？

**Reply by Geolage** [reply]
刚看了一下 我没有设置endpoint的 但为啥还是会经过服务器呢

**Reply by 1715173329** [reply]
我不清楚你 Android 客户端怎么配的，可以考虑用杜比大喇叭。

**Reply by Geolage** [reply]
sorry 搞错了 是PC版的 安卓端的我一直用着dolby 没啥问题

**Reply by Geolage** [reply]
主要是这个缓冲速度太慢了 感觉跟我服务器的带宽基本一致 不解为什么会走服务器而不是客户端拿到资源链接后直接请求
